examples/SimpleToDoAvalonia/SimpleToDo.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>net7.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
+        <ApplicationManifest>app.manifest</ApplicationManifest>
+        <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Folder Include="Models\" />
+        <AvaloniaResource Include="Assets\**" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <TrimmerRootAssembly Include="Avalonia.Themes.Fluent" />
```

Comment:
This doesn't seem to be committed.

Reply:
Good catch! I didn't look too much at the default project file

Reply:
The assembly?

Reply:
Aaah I see. I've actually been using Rider with this, since it was also what Avalonia recommended, and that doesn't show anything in the solution explorer. I suspect that is a VS Mac bug. 
I would prefer not to remove that line from the project file, given that it was given by the Avalonia template. They could be doing something dynamic with that assembly and that's why it was supposed to not be trimmed, so I wouldn't risk it 😁

Reply:
Sounds good!